app/main.py

```diff
@@ -1 +1,45 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    @classmethod
+    def kill(cls, animal: Herbivore) -> None:
```

Comment:
The `kill` method is defined to accept a `Herbivore` instance, but since it's a class method of `Animal`, it might be more appropriate to accept an `Animal` instance. This would allow it to be used for any animal, not just herbivores.